prime-router/src/test/kotlin/fhirengine/utils/FHIRBundleHelpersTests.kt

```diff
@@ -336,6 +203,7 @@ class FHIRBundleHelpersTests {
         assertThat(bundle.getDiagnosticReportNoObservations()).isEmpty()
     }
 
+    // TODO: fix OOM exception, ticket:
```

Comment:
What's the plan here? Wasn't sure if this test needs to be commented out or something in the meantime.

Reply:
Looking again, I guess this didn't need to be skipped after increasing the heap size? But we still want a ticket to look for unnecessary memory usage?

Reply:
Yep... I'll still add a ticket